Formula/i/ibazel.rb

```diff
@@ -16,14 +17,7 @@ class Ibazel < Formula
   end
 
   depends_on "go" => [:build, :test]
-
-  on_macos do
-    depends_on "bazel" => [:build, :test]
-  end
-
-  on_linux do
-    depends_on "bazelisk" => [:build, :test]
-  end
+  depends_on "bazelisk" => [:build, :test]
```

Comment:
Switched to `bazelisk` as the build dep for `macos` as well here, so that it's easy to build it with the less-than-latest Bazel version we have in brew.

Reply:
good call 👍